fitbenchmarking/templates/js/table.js

```diff
@@ -0,0 +1,65 @@
+/**
+* Uses jQuery to find a collection of DOM elements with text equal to the provided search text.
+* @param   {String} class_name    The name of a html element class to find.
+* @param   {String} search_text   The text to search for.
+* @return  {jQuery Object}        A collection of DOM elements which matched the jQuery search.
+*/
+function _find_element_from_text(class_name, search_text) { 
+    return $(class_name).filter(function() { return $(this).text() == search_text; });
+}
+
+/**
+* Adjusts the span of a column header that encompasses a number of sub-headers.
+* @param   {jQuery Object} header   A collection of DOM elements representing column headers with a span.
+* @param   {bool} increment         True if the column span should be incremented. Otherwise it is decremented.
+*/
+function _adjust_colspan(header, increment) {
+    var colspan = parseInt(header.attr('colspan'));
+    var new_colspan = increment ? colspan + 1: colspan - 1;
+    header.attr('colspan', new_colspan);
+
+    // Hide the header if the sub-headers are all hidden
+    if (new_colspan == 0 || isNaN(new_colspan)) {
```

Comment:
This now fails on show as nan+1=nan so it stays hidden.
We may have to do something like this on before `var new_colspan` (syntax may be wrong):
```
if (isNaN(colspan)) {
    colspan = 1;
}
```

Reply:
Thanks again! Should be working this time round